.circleci/config.yml

```diff
@@ -64,6 +64,8 @@ jobs:
           no_output_timeout: "30m"
       - store_artifacts:
           path: /go/src/github.com/Azure/aks-engine/_logs
+      - store_artifacts:
```

Comment:
Why do we need this?  We had removed this as a part of this PR - https://github.com/Azure/acs-engine/pull/4214

Reply:
I added this for debug purposes.

Why did you remove it ?

Reply:
let's revert adding this back - circle ci is not used anymore

Reply:
ok